src/receiveEndpoint.ts

```diff
@@ -119,6 +121,10 @@ export class ReceiveEndpoint extends Transport implements ReceiveEndpointConfigu
 
         await channel.bindQueue(this.queueName, this.queueName, '');
 
+        for (const messageType of this.boundEvents) {
+            await channel.bindExchange(this.queueName, messageType.toExchange(), '');
+        }
+
```

Comment:
can we get a test on this?

Reply:
Added tests for the `configureTopology` method.